src/test/scala/chiselTests/stage/ChiselOptionsViewSpec.scala

```diff
@@ -15,7 +16,7 @@ class ChiselOptionsViewSpec extends AnyFlatSpec with Matchers {
   behavior of ChiselOptionsView.getClass.getName
 
   it should "construct a view from an AnnotationSeq" in {
-    val bar = Circuit("bar", Seq.empty, Seq.empty)
+    val bar = Circuit("bar", Seq.empty, Seq.empty, RenameMap())
```

Comment:
I'm kind of surprised this was the only place you had to update `Circuit`

Reply:
Me too, but yay type-safety!